integrationTests/cypress/integration/02-apd/01-apd-basics.spec.js

```diff
@@ -347,6 +348,7 @@ describe('APD Basics', { tags: ['@apd', '@default'] }, () => {
         .check({ force: true });
       cy.findByRole('button', { name: /Save/i }).click();
 
+      cy.log("LOL")
```

Comment:
do you still need all of these log statements

Reply:
Haha, nope. Removing.